pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -20,25 +20,60 @@ This section describes how to upgrade your Kommander Management cluster and all
 -   Review the [Platform Application version updates][release_notes] that are part of this upgrade.  
 -   For air-gapped environments **with** DKP Catalog Applications in a multi-cluster environment: [Load the Docker images into your Docker registry][load_images_catalog]
 -   For air-gapped environments **without** DKP Catalog Applications: [Load the Docker images into your Docker registry][load_images]
--   For air-gapped environments only:
-  
+
   Download the Kommander application definitions:
 
   ```bash
   wget "https://downloads.d2iq.com/dkp/v2.2.0/kommander-applications-v2.2.0.tar.gz"
   ```
-  
+
   Download the Kommander charts bundle:
 
   ```bash
   wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-kommander-charts-bundle-v2.2.0.tar.gz"
   ```
-  
+
   If you have any DKP Catalog Applications, download the DKP Catalog Application charts bundle:
 
   ```bash
   wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-catalog-applications-charts-bundle-v2.2.0.tar.gz"
   ```
+  
+  <p class="message--note"><strong>NOTE:</strong> Beginning with DKP version 2.2, MetalLB is no longer managed as a catalog application. If you installed MetalLB on the cluster that you're upgrading prior to DKP version 2.2, you will need to detach MetalLB from the cluster prior to upgrading.</p>
```

Comment:
```suggestion
  <p class="message--important"><strong>NOTE:</strong> Beginning with DKP version 2.2, MetalLB is no longer managed as a platform application. If you installed MetalLB on the cluster that you're upgrading prior to DKP version 2.2, you will need to detach MetalLB from the cluster prior to upgrading.</p>
```

Might bump this one up to an important

Reply:
Maybe we can include this section under a subheading to further highlight these steps?